example/demo/README.md

```diff
@@ -0,0 +1,82 @@
+# Debezium UI Example
+
+This demo automatically deploys the topology of services as defined in the [Debezium Tutorial](https://debezium.io/docs/tutorial/) along with Debezium UI.
```

Comment:
Let's link to the right URL, not go through the redirect.

Reply:
sure